manifests/cf-manifest/cloud-config/010-cf-networks.yml

```diff
@@ -1,7 +1,6 @@
 ---
-
 networks:
-- name: cf1
+- name: cf
   subnets:
     - range: 10.0.16.0/24
       reserved:
```

Comment:
New commit sounds good. It's a perfect opportunity now to fix such small things.